mobile-2/App.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { StyleSheet, Text, View, Image, ScrollView } from 'react-native';
+import { SearchBar } from 'react-native-elements';
+
+export default class App extends React.Component {
+	render() {
+		let pic = {
```

Comment:
In this case, you can create these images in a separate file and just import them into your component.